lib/parsers/static_text_parser.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const Types = require('../constants/types.js');
+const Charsets = require('../constants/charsets.js');
+const helpers = require('../helpers');
+
+const typeNames = [];
+for (const t in Types) {
+  typeNames[Types[t]] = t;
+}
+
+function readField({ packet, type, charset, encoding, config, options }) {
+  const supportBigNumbers =
+    options.supportBigNumbers || config.supportBigNumbers;
+  const bigNumberStrings = options.bigNumberStrings || config.bigNumberStrings;
```

Comment:
```suggestion
  const supportBigNumbers =
    Boolean(options.supportBigNumbers || config.supportBigNumbers);
  const bigNumberStrings = Boolean(options.bigNumberStrings || config.bigNumberStrings);
```

Preventing a potential **RCE**, see #2572.